INSTALL.md

```diff
@@ -0,0 +1,71 @@
+# Installation
+
+The subscription-manager codebase is on [GitHub](http://github.com/candlepin/subscription-manager).
+The project is built for the latest versions of Fedora (and submitted to Fedora Updates), CentOS Stream and RHEL.
+
+To install subscription-manager, run
+
+```bash
+sudo dnf install subscription-manager
+```
+
+## Developer installation
+
+The process below has been tested on Fedora 36.
+Other Fedora versions or distributions may require some adaptation.
+
+1. First you need to install RPM packages to build and run subscription-manager binaries:
+
+   ```bash
+   sudo dnf install git gcc python3-devel openssl-devel intltool dbus-devel glib2-devel python3-rpm python3-librepo libnl3-devel
```

Comment:
hmm we don't use neither intltool nor libnl3-devel nor dbus-devel... check the `BuildRequires` line unconditionally used (ie not for dnf/libdnf/zypper) in the spec file

... and libnotify-devel is obsolete, I just created #3111 for dropping it

Reply:
On the other side. I miss `libdnf-devel`. It is not possible to build libdnf plugin `product-id` without that. If you want to build this plugin, then you also need `cmake`.

Reply:
considering that there are python3 modules mentioned before, I think that a safe option for people to test sub-man can be installing all the python modules from the distro, and not mention pip at all (leaving it only to people that are comfortable with it).